### PR TITLE
Sentry: support TLS for Redis

### PR DESCRIPTION
**Pull request by maciejp-ro**
This builds on top of import from #20 and makes it possible to configure a TLS connection to Redis

### CODE REVIEW

File: charts/sentry/Chart.yaml [hunk 1]
```diff
@@ -39,4 +39,4 @@ maintainers:
 - name: sentry-kubernetes
 name: sentry
 type: application
-version: 19.4.0
+version: 19.4.0+1
```
Comment (michaeljguarino) [inline charts/sentry/Chart.yaml]: i'm not totally sure how helm will take to that versioning but we can check

**Review by michaeljguarino — APPROVED** [review]

File: charts/sentry/Chart.yaml (hunk 1 above)
Comment (maciejp-ro) [reply charts/sentry/Chart.yaml]: Helm doesn't complain (docs say it uses semver 2.0, which specifies `-prerelease` and `+metadata` suffix). I used metadata to indicate local changes.

Linter complains about it, though, and I checked semver spec - it says that metadata should be ignored when comparing versions (it's for things like build id). Using prerelease suffix (`9.4.0-plural1`) might calm the linter down. I'm on mobile now but feel free to push an update to check - I allowed maintainer pushes on this PR

Comment (maciejp-ro) [reply charts/sentry/Chart.yaml]: Managed to push the change, linter's still unhappy - and AFAIR Helm doesn't like it when you try to deploy changed chart without a version bump. I'm not sure how to proceed here, feel free to update this branch

Comment (michaeljguarino) [reply charts/sentry/Chart.yaml]: I think it's probably fine, just wanted to confirm helm won't bork on it

**Pull request merged**